src/lib/Libutil/misc_utils.c

```diff
@@ -976,7 +1041,6 @@ validate_ext_auth_data(int auth_type, void *data, int data_len, char *ebuf, int
 char **
 break_comma_list(char *strlist)
 {
-	static char id[] = "break_comma_list";
```

Comment:
`static char id[]` which used to hold the function names are replaced by `__func__`. So this is no longer relevant.